video/out/opengl/hwdec_cuda.c

```diff
@@ -302,30 +313,42 @@ static struct ra_buf *cuda_buf_pool_get(struct ra_hwdec_mapper *mapper, int n)
 
     if (!ra_vk_buf_get_user_data(buf)) {
         struct ext_buf *ebuf = talloc_zero(NULL, struct ext_buf);
+        struct vk_external_mem mem_info;
 
-        struct vk_external_fd fd_info;
-        bool success = ra_vk_buf_get_external_fd(mapper->ra, buf, &fd_info);
+        bool success = ra_vk_buf_get_external_info(mapper->ra, buf, &mem_info);
         if (!success) {
             ret = -1;
             goto error;
         }
 
-        ebuf->fd = fd_info.mem_fd;
-        MP_DBG(mapper, "vk_external_fd[%d][%d]: %d %lu %lu\n", n, pool->index, ebuf->fd, fd_info.size, fd_info.offset);
+#if HAVE_WIN32_DESKTOP
+        MP_DBG(mapper, "vk_external_info[%d][%d]: %p %zu %zu\n", n, pool->index, ebuf->handle, mem_info.size, mem_info.offset);
+        ebuf->handle = mem_info.mem_handle;
+#else
+        MP_DBG(mapper, "vk_external_info[%d][%d]: %d %zu %zu\n", n, pool->index, ebuf->fd, mem_info.size, mem_info.offset);
+        ebuf->fd = mem_info.mem_fd;
+#endif
```

Comment:
You changed the order here, the debug print will thus never print the handle/fd.